pyccel/codegen/printing/pycode.py

```diff
@@ -852,6 +852,14 @@ def _print_ListAppend(self, expr):
 
         return f"{list_var}.{method_name}({append_arg})\n"
 
+    def _print_ListInsert(self, expr):
+        method_name = expr.name
+        index = self._print(expr.index)
+        list_var = self._print(expr.list_variable)
+        insert_arg = self._print(expr.insert_argument)
+
+        return f"{list_var}.{method_name}({index},{insert_arg})\n"
```

Comment:
For readability and as a convention in most Python codes spaces are usually used after commas
```suggestion
        return f"{list_var}.{method_name}({index}, {insert_arg})\n"
```

Reply:
updated in cd5067e8f92adfe31a66bee6e69028e6d7ef56a0